src/app/containers/CpsFeaturesAnalysis/index.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { useContext } from 'react';
+import { arrayOf, shape } from 'prop-types';
+import { StoryPromoLi, StoryPromoUl } from '@bbc/psammead-story-promo-list';
+
+import featuresAnalysis from '#pages/StoryPage/featuresAnalysis.json';
+import { storyItem } from '#models/propTypes/storyItem';
+import { ServiceContext } from '#contexts/ServiceContext';
+import StoryPromo from '../StoryPromo';
+import CpsAsset from '../CpsAsset';
+
+const FeaturesAnalysis = ({ content }) => {
+  const { dir } = useContext(ServiceContext);
+  const a11yAttributes = {
+    as: 'section',
+    role: 'region',
+    'aria-labelledby': 'features-analysis-heading',
+  };
+
+  if (!content.length) return null;
```

Comment:
this same logic is still duplicated across `FeaturesAnalysis`, `CpsRelatedContent` and `CpsTopStories`, surely if we aim to make this a DRY approach, shouldn't it be encompassed within CpsAsset ?

Reply:
Thanks. I've taken out the line from the containers' code.